src/Logikfabrik.Umbraco.Jet/Extensions/PreValueCollectionExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Logikfabrik.Umbraco.Jet.Extensions
```

Comment:
Is there a need for this function to be an extension method? I'm thinking that since it's only used within `DataTypeSynchronizer` it could be moved inside that class instead?